src/transformers/__init__.py

```diff
@@ -2049,17 +2070,6 @@
             "load_tf_weights_in_t5",
         ]
     )
-    _import_structure["models.switch_transformers"].extend(
```

Comment:
I put this in the appropriate alphabetical place

Reply:
It already is. Not sure what branch you are on but a rebase/merge seems necessary.